### ISSUE TITLE
JSQ can scroll horizontally on first appear(which should not)

### ISSUE THREAD

**Issue by TaoistKing** [jessesquires/JSQMessagesViewController#1917]
## New issue checklist

- [YES ] I have read all of the [`README`]
- [YES ] I have reviewed the [contributing guidelines]
- [YES ] I have searched [existing issues]
 **this is not a duplicate**.

## General information

- Library version(s): 7.3.3
- iOS version(s): 10.0.2
- Devices/Simulators affected:
- Reproducible in the demo project? (Yes/No): No
- Related issues:

## What happened?
Hi, I'm building chat app with JSQ, it works well on iPhone6/6P, but when running on a iPhone5/5c devices(smaller screen), the viewController shows peer avatar on view first appeared, the local avatar doesn't shows up until a horizontal scroll to right, then after sending or receiving a message, both avatar appears and horizontal scroll is disabled.

**Comment by TaoistKing** [comment]
Please refer the gif for details.
Upon view first appeared, view is horizontal scrollable(and local avatar not showing), after sending a message(or collectionView reload), the view fits the screen and both avatar shows up. Please help.

![11 -23-2016 12-56-04](https://cloud.githubusercontent.com/assets/10893482/20551564/7c1d2d64-b17c-11e6-8763-2b3d301dde1b.gif)

**Comment by TaoistKing** [comment]
Hi, After more debugging, I solved this problem by kind of hack.

I found that collectionViewLayout:sizeForItemAtIndexPath  will return a width of 367 after viewDidLoad, but return a width of 312 after ViewWillAppear.
(width of 312 fits iPhone 5 screen, 367 makes local avatar outside the screen)
(and that's why sending a message make the view fits again)

So What I did is return Nil messages before ViewWillAppear, 
Until ViewWillAppear I will load my messages and call `[self.collectionView reloadData]`

But I have No idea of why collectionViewLayout would return different cell width before and after ViewWillAppear.
